tuned/plugins/plugin_modules.py

```diff
@@ -49,12 +49,15 @@ def _instance_apply_static(self, instance):
 			module = self._variables.expand(option)
 			v = self._variables.expand(value)
 			if not skip_check:
-				retcode, out = self._cmd.execute(["modinfo", module])
+				# modinfo could fail as expected when module is not
+				# there. Capture the error instead of printing an
+				# ERROR every time even it's a very valid case
+				retcode, out, err = self._cmd.execute(["modinfo", module], return_err=True)
 				if retcode < 0:
 					skip_check = True
-					log.warn("'modinfo' command not found, not checking kernel modules")
+					log.info("'modinfo' command not found, not checking kernel modules")
 				elif retcode > 0:
-					log.error("kernel module '%s' not found, skipping it" % module)
+					log.info("kernel module '%s' not found, skipping it" % module)
```

Comment:
If a kernel module is not found, it can be a real problem (e.g. the module name might be misspelled in the Tuned profile, or the admin forgot to build the required kernel module). So I don't think it makes sense to hide this message. It might make sense to change it to `log.warn`, though.

It might make sense to merge the message printed by `self._cmd.execute` and by the line above into one message, so that we don't print two messages regarding one problem unnecessarily. I think we could do something similar to the following here:
`log.warn("modinfo failed, skipping the module '%s': %s" % (module, err))`
so that the message becomes something like
`WARNING ... modinfo failed, skipping the module 'cpufreq_conservative': modinfo: ERROR: Module cpufreq_conservative not found.`

... but that's getting a bit long, isn't it?

Anyway, note that the error message
`Executing modinfo error: modinfo: ERROR: Module cpufreq_conservative not found.`
in issue #192 can IMO be considered misconfiguration in the `balanced` profile. We're going to fix the error another way - we'll load the `cpufreq_conservative` module in the `cpu` plugin, if necessary, instead of doing it through the `modules` plugin.